### ISSUE TITLE
I want to download data from website, but there is something wrong with link https://challenger.ai/competition/wf2018.

### ISSUE THREAD

**Issue by xyk0058** [BruceBinBoxing/Deep_Learning_Weather_Forecasting#1]

**Comment by BruceBinBoxing** [comment]
I noticed that the competition link has been closed. However, competitors have uploaded the [raw dataset](https://github.com/Ericonaldo/AI-Challenger-Weather-Forecast/tree/master/data).
You can use our  script to preprocessing below datasets:

Training set: 
  ai_challenger_wf2018_trainingset_20150301-20180531.nc
Validation set: 
  ai_challenger_wf2018_validation_20180601-20180828_20180905.nc
Test set: 
  1) ai_challenger_wf2018_testa1_20180829-20180924.nc
  2) ai_challenger_weather_testingsetB_20180829-20181015.nc
  3) ai_challenger_wf2018_testb1_20180829-20181028.nc
  4) ai_challenger_wf2018_testb2_20180829-20181029.nc
  5) ai_challenger_wf2018_testb3_20180829-20181030.nc
  6) ai_challenger_wf2018_testb4_20180829-20181031.nc
  7) ai_challenger_wf2018_testb5_20180829-20181101.nc
  8) ai_challenger_wf2018_testb6_20180829-20181102.nc
  9) ai_challenger_wf2018_testb7_20180829-20181103.nc

Best.

**Comment by xyk0058** [comment]
Thanks!
